e2e/test/scenarios/permissions/sandboxing/helpers/e2e-sandboxing-helpers.ts

```diff
@@ -317,6 +322,21 @@ export function rowsShouldContainOnlyGizmos(responses: DatasetResponse[]) {
   ).to.be.true;
 }
 
+export const valuesShouldContainGizmosAndWidgets = (
+  valuesArray: (FieldValue | ParameterValue)[],
+) => {
+  const values = valuesArray.map(val => val[0]);
+  expect(values).to.contain("Gizmo");
+  expect(values).to.contain("Widget");
+};
+
+export const valuesShouldContainOnlyGizmos = (
+  valuesArray: (FieldValue | ParameterValue)[],
+) => {
+  const values = valuesArray.map(val => val[0]);
+  expect(values).to.deep.equal([["Gizmo"]]);
+};
+
```

Comment:
Since the assertions we want to make about parameter values are very similar to the ones we want to make about field values, I've refactored the field value assertion helper to make it just like the parameter value assertion helper. This keeps the code cleaner in the tests.